javascript/code-challenges/challenges-03.test.js

```diff
@@ -0,0 +1,311 @@
+'use strict';
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 1 - Review
+
+Write a function called addTwo that takes in an array and adds two to every value using a for loop. Place the new value in a new array. Return the new array.
+------------------------------------------------------------------------------------------------ */
+
+const addTwo = (arr) => {
+  let newArr = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    let newSum = arr[i] + 2;
+    newArr.push(newSum);
+  }
+  return newArr;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 2
+
+Write a function named typeNum that, given an array as input, uses filter to return an array containing only the numbers.
+
+For example, typeNum([1, 'bob' ,3]) returns [1,3].
+------------------------------------------------------------------------------------------------ */
+
+const typeNum = (arr) => {
+  let numOnly = (val) => {
+    if(typeof (val) === 'number') {
+      return val;
+    }
+  };
+  let numbers = arr.filter(numOnly);
+  return numbers;
+};
+
+/* ------------------------------------------------------------------------------------------------
+CHALLENGE 3
+
+Write a function named containsAnd that, given an array of strings as input, uses filter to return an array containing only strings that contain 'and' within the string.
+
+For example, containsAnd(['panda', 'ran', 'and']) returns ['panda', 'and'].
+------------------------------------------------------------------------------------------------ */
+
+const containsAnd = (arr) => {
+  var charc = arr.filter(charInc => charInc.includes('and'));
+  return charc;
+};
```

Comment:
You can just have your "var" as a return value.
Also, "var" is a depreciated. You should always use either "let" or "const".